applications/index.md

```diff
@@ -68,6 +68,8 @@ menu_item: true
     is a LADISH frontend.
   * [**Carla (KXStudio)**](http://kxstudio.sourceforge.net/Applications:Carla)
     is an audio plugin host.
+  * [**Jack Sanit**](https://github.com/psychoticmeow/jack-sanity)
```

Comment:
isn't it called "Jack Sanity" you may want to correct that typo in the name of your own app ;)